### PR TITLE
Setup cob4-10

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Please merge first the cob4-7 setup PR https://github.com/ipa320/cob_robots/pull/481

**Comment by ipa-fxm** [comment]
rebase to get rid of ```cob4-7```-related commits

**Review by fmessmer — CHANGES_REQUESTED** [review]
rebase to get rid of ```cob4-7```-related commits and merge-conflicts

File: cob_hardware_config/cob4-10/config/teleop.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#cob4-7
```
Comment (fmessmer) [inline cob_hardware_config/cob4-10/config/teleop.yaml]: ```#cob4-10```

File: cob_hardware_config/cob4-10/urdf/cob4-10.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
+       name="cob4-7" >
```
Comment (fmessmer) [inline cob_hardware_config/cob4-10/urdf/cob4-10.urdf.xacro]: ```cob4-10```

File: cob_hardware_config/cob4-10/config/teleop.yaml (hunk 1 above)
Comment (NadiaHG) [reply cob_hardware_config/cob4-10/config/teleop.yaml]: done!

File: cob_hardware_config/cob4-10/urdf/cob4-10.urdf.xacro (hunk 2 above)
Comment (NadiaHG) [reply cob_hardware_config/cob4-10/urdf/cob4-10.urdf.xacro]: done!

**Review by fmessmer — APPROVED** [review]

**Comment by fmessmer** [comment]
I'm going to squash this PR into a single commit...

**Pull request merged**